src/consts.ts

```diff
@@ -12,3 +12,11 @@ export const APPLET_PAGES = {
 };
 
```

Comment:
I think, you shouldn't rename and move this file

Reply:
utils is not the right place for constants

Reply:
and I renamed because with constants some problem with import when constants.ts